packages/mgt-components/src/components/mgt-todo/mgt-todo.scss

```diff
@@ -6,12 +6,131 @@
  */
 
 @import '../mgt-tasks-base/mgt-tasks-base.scss';
+@import '../mgt-tasks-base/mgt-tasks-base.theme.scss';
 
 :host {
   @extend :host;
   border-radius: 8px;
   .Task {
     @extend .Task;
-    border: none;
+    width: auto;
```

Comment:
@gavinbarron This only works for storybook, but the developer test page is still weird, for the add task input

Reply:
The issue persisted even with setting that to auto but giving it a max-width similar to the fluent-text-box width seems to work.

Reply:
We shouldn't be setting explicit widths as a rule of thumb. It makes it really hard to integrate our component into a web page that might use a layout that is narrower than the explicit width we have